slices/apache2-bin.yaml

```diff
@@ -0,0 +1,36 @@
+package: apache2-bin
+
+essential:
+  - apache2-bin_copyright
+
+slices:
+  bins:
+    essential:
+      - apache2-bin_modules
+    contents:
+      /usr/sbin/apache2:
```

Comment:
I think we should add some spread tests for `/usr/sbin/apache2` in a new `tests/spread/integration/apache2-bin/task.yaml`.